.github/dependabot.yml

```
@@ -1,25 +1,25 @@
+    commit-message:
+      prefix: "deps"
+    open-pull-requests-limit: 5
+    target-branch: "main"
+    groups:
+      all:
+        patterns: ["*"]
+    ignore:
+      # Ignore major version updates for stability
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major"]
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)